src/styles.js

```diff
@@ -0,0 +1,137 @@
+import styled from 'styled-components';
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  html, body {
+    height: 100%;
+  }
+
+  body {
+    font-family: Arial, sans-serif;
+    color: black;
+  }
+`;
+
+export const MainText = styled.div`
+font-size: 50px;
+font-weight: 600;
+`
+
+export const Line = styled.div`
+background-color: black;
+width: 1.5px;
+height: 100px;
+margin-bottom: 10px;
+`
+
+export const AppContainer = styled.div`
+  margin-left: 70px;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  flex-direction: column;
+`;
+
+export const Main = styled.main`
+  height: 400px;
+  width: 400px;
+`;
+
+
+export const TodoContainer = styled.section`
+  height: 400px;
+  border-color: black;
+  border-radius: 10px;
+  border-width: 2px;
+  border-style: solid;
+  padding: 20px;
+`;
+
+export const TodoList = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin-bottom: 10px;
+`;
+
+export const TodoItem = styled.li`
+  background-color: aliceblue;
+  padding: 10px;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  display: flex;
+  justify-content: space-between;
+  word-break: break-all;
+`;
+
+export const TodoInput = styled.div`
+  display: flex;
+  justify-content: space-between;
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 1px;
+  margin-bottom: 10px;
+`;
+
+export const Input = styled.input`
+  flex-grow: 1;
+  padding: 10px;
+  border-radius: 7px;
+  border: none;
+  margin-right: 10px;
+  background-color: transparent;
+  border-radius: 10px;
+`;
+
+export const TodosBody = styled.div`
+  height: 250px;
+  border-radius: 5px;
+  overflow-y: auto;
+  &::-webkit-scrollbar {
+    width: 8px;
+    height: 8px;
+    border-radius: 6px;
+    background: rgba(255, 255, 255, 0.4);
+  }
+  &::-webkit-scrollbar-thumb {
+    background: rgba(0, 0, 0, 0.3);
+    border-radius: 6px;
+  }
+`
+
+export const Button = styled.button`
+  padding: 8px 8px;
+  background-color: skyblue;
+  border: none;
+  border-radius: 5px;
```

Comment:
<img width="68" alt="스크린샷 2024-09-23 오후 5 37 36" src="https://github.com/user-attachments/assets/40a166b9-76aa-4df1-91a5-a4e6bfee7498">
<img width="56" alt="스크린샷 2024-09-23 오후 5 37 43" src="https://github.com/user-attachments/assets/784ed9c3-2746-4686-8ec5-2c1f7a3e9327">

여기도 겹쳐지는 부분에 `border-radius` 값를 통일시켜주기로 해요!
```suggestion
  border-radius: 10px;
```